lib/appengine/tasks.rb

```diff
@@ -20,6 +20,9 @@
 require "appengine/util/gcloud"
 require "appengine/exec"
 
+require "google/serverless/exec/gcloud"
+require "google/serverless/exec"
```

Comment:
Put these require statements in the appengine/exec.rb file because that's where they're actually used. (i.e. that's where you're referencing the `Google::Serverless::Exec` class.) Also, you shouldn't need to require google/serverless/exec/gcloud directly, since google/serverless/exec.rb already requires it.